tests/Feature/PhotosTest.php

```diff
@@ -35,7 +35,7 @@ public function testUpload()
 		$file = new UploadedFile(
 			'public/uploads/import/night.jpg',
 			'night.jpg',
-			'image/jpg',
```

Comment:
it is weird that you need to modify that one. :thinking:

Reply:
I thought that may have been the problem but turns out it wasn't. It was a failing test leaving a leftover photo in the database and affecting the next case. Still, `image/jpg` is not a valid mime type. It should say `image/jpeg`. So I think it's still worth fixing.